app/javascript/packs/shared/navigation/components/GlobalActivitiesModal.jsx

```diff
@@ -4,10 +4,46 @@ import PropTypes from "prop-types";
 import { FormattedMessage } from "react-intl";
 import { Modal, Button } from "react-bootstrap";
 import _ from "lodash";
+import styled from "styled-components";
 
 import { getActivities } from "../../actions/ActivitiesActions";
 import ActivityElement from "./ActivityElement";
 import ActivityDateElement from "./ActivityDateElement";
+import {
+  WHITE_COLOR,
+  COLOR_CONCRETE,
+  COLOR_MINE_SHAFT,
+  COLOR_GRAY_LIGHT_YADCF
+} from "../../../app/constants/colors";
+
+const StyledBottom = styled(Button)`
+  display: inline-block;
+  margin-bottom: 0;
+  font-weight: normal;
+  text-align: center;
+  vertical-align: middle;
+  touch-action: manipulation;
+  cursor: pointer;
+  background-image: none;
+  border: 1px solid transparent;
+  white-space: nowrap;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857143;
+  border-radius: 4px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
I think autoprefixing is done automatically by styled-components

https://github.com/styled-components/styled-components/pull/54